### ISSUE TITLE
New Bernoulli get "Object reference not set to an instance of an object."

### ISSUE THREAD

**Issue by ChengYen-Tang** [dotnet/TorchSharp#838]
Python:
```
>>> from torch.distributions import Categorical
>>> import torch as th
>>> th.manual_seed(1)
>>> nSamples = int(5e6)
>>> actionLogits = th.rand(nSamples, 2)
>>> distribution = Bernoulli(logits=actionLogits)
```

C#
```
using TorchSharp;
using TorchSharp.Modules;
using static TorchSharp.torch;

int nSamples = Convert.ToInt32(5e6);
random.manual_seed(1);
Tensor actionLogits = rand(nSamples, 2);
Bernoulli distribution = new(l: actionLogits);
# System.NullReferenceException: 'Object reference not set to an instance of an object.'
```

**Comment by NiklasGustafsson** [comment]
@ChengYen-Tang -- thanks for your bug reports, and for the small repos you posted. That always helps me get to a resolution faster.

**Comment by ChengYen-Tang** [comment]
Hope this project is more perfect.

**Comment by NiklasGustafsson** [comment]
I have verified the bug. Unfortunately, it will need a version update on NuGet in order to be addressed. In the meantime, you can work around it by using probs instead of logits to create the distribution. Not sure that works for you, but that's the current workaround.

**Comment by ChengYen-Tang** [comment]
Thanks, I can develop other things first and wait for the package update
